app/budget/directives/courseCosts/courseCosts.css

```diff
@@ -34,22 +34,24 @@
  {
 	display: flex;
 	flex: 5 0 0px;
-	padding: 15px;
+	padding: 19px;
+	border-bottom: 1px solid #ebebeb;
 }
 
 .course-costs .cell {
 	flex: 1 0 0px;
 	display: flex;
 	align-items: center;
 	justify-content: center;
+	/*background-color: #F9F9F9;*/
 }
```

Comment:
If you don't want this style anymore, delete it! Commented code is very rarely a good thing in a commit.